### PR TITLE
Add issue template for kafka topics/users

### PR DESCRIPTION
**Pull request by 4n4nd**
Resolves https://github.com/operate-first/apps/issues/805

Can be seen what it will look like here: https://github.com/4n4nd/odh-moc-support/issues/new?assignees=&labels=user-support&template=kafka_user_topic_request.md&title=

### CODE REVIEW

**Comment by HumairAK** [comment]
This is great, but can you hide the section descriptions like we do it here: https://github.com/operate-first/support/blob/main/.github/ISSUE_TEMPLATE/onboarding_to_cluster.md 

I'd like to start doing this for all templates.

**Comment by 4n4nd** [comment]
@HumairAK updated :+1:

**Review by harshad16 — COMMENTED** [review]
/lgtm
content wise it seems very apt
i don't know how the <!-- would work in the issue template , so please check at in your fork

File: .github/ISSUE_TEMPLATE/kafka_user_topic_request.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+   <!--
+   A list of kafka topic names to be requested.
+   -->
+
+2. **Topic partitions**:
+   <!--
+   Number of topic partitions.
+   -->
+
+   Default: 3
```
Comment (HumairAK) [inline .github/ISSUE_TEMPLATE/kafka_user_topic_request.md:23]: Can you move the `Default: 3` into the hidden portion and add: 

PARTITIONS: 

And do the same for the ones below ?

Comment (4n4nd) [reply .github/ISSUE_TEMPLATE/kafka_user_topic_request.md:23]: I think it looks cleaner this way

Comment (4n4nd) [reply .github/ISSUE_TEMPLATE/kafka_user_topic_request.md:22]: tried something else

**Pull request merged**